src/components/PDFViewer/PDFViewer.scss

```diff
@@ -4,6 +4,7 @@
   background-color: #f2f8fb;
 }
 .pdf-title{
+  max-width: 355px;
```

Comment:
Is this supposed to be a min-width?

Reply:
oops that was just a mistake i'll take it out